src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { Flex, color, Box, ResponsiveImage, Sans } from "@artsy/palette"
+import styled from "styled-components"
+import { NavBarHeight } from "Components/NavBar"
+
+export const ViewingRoomHeader: React.FC = props => {
+  return (
+    <HeaderContainer>
+      <ImageArea>
+        <ResponsiveImage
+          src="https://user-images.githubusercontent.com/236943/81243255-342dcc00-8fc4-11ea-9a2b-2ab96ab67c9b.png"
+          style={{
+            backgroundSize: "cover",
+            backgroundPosition: "center",
+            height: "100%",
+          }}
+        />
```

Comment:
We gotta fix the ResponsiveImage component to not use background images so we can support proper srcSet and (and it's just annoying)

Reply:
[Here's the one I did recently](https://github.com/auspices/eos/blob/master/src/ResponsiveImage/ResponsiveImage.tsx)

Reply:
Its been super super annoying through time...